src/catkin_pkg/package.py

```diff
@@ -370,6 +371,14 @@ def __str__(self):
         return txt
 
 
+# Subclassing ``str`` to keep backward compatibility.
+class License(str):
+    def __new__(cls, value, path=''):
```

Comment:
The default value should be `None` in order to be distinguishable from an existing attribute with an empty string value.

Reply:
I will change the default value is None.